src/libstd/keyword_docs.rs

```diff
@@ -1030,22 +1030,39 @@ mod self_upper_keyword {}
 //
 /// A place that is valid for the duration of a program.
 ///
-/// A `static` item is similar to a [`const`] item in that it lives for the
-/// entire duration of the program and need to have its type explicited, with a
-/// `static` lifetime, outliving any other lifetime. Added to that, `static`
-/// items represent a precise memory location.
+/// A static item is a value which is valid for the entire duration of your
+/// program (a `'static` lifetime).
```

Comment:
This sentence was supposed to replace "A place that is valid for the duration of a program." 
So just remove that old sentence :)